Source/WebKit/Platform/Logging.h

```diff
@@ -51,6 +51,7 @@ extern "C" {
     M(CacheStorage) \
     M(ContentObservation) \
     M(ContextMenu) \
+    M(CoordinatedGraphics) \
```

Comment:
Are we really using this new logging channel in the end?

Reply:
No, it's a leftover from an older version of the patch. I'll remove that.